HeterogeneousCore/AlpakaUtilities/BuildFile.xml

```diff
@@ -0,0 +1,13 @@
+<use name="rootcore"/>
+<use name="boost"/>
+<use name="eigen"/>
+<use name="DataFormats/Common"/>
+<use name="DataFormats/Portable"/>
+<use name="DataFormats/SoATemplate"/>
+<use name="HeterogeneousCore/AlpakaInterface"/>
+<flags ALPAKA_BACKENDS="1"/>
```

Comment:
The `ALPAKA_BACKENDS` flag is not needed either as there is no code making use of `ALPAKA_ACCELERATOR_NAMESPACE`.

Reply:
I wonder if that even does anything, for files that are just header files and are not inside the `alpaka/` subdirectory ?